packages/devtools_app/lib/src/screens/memory/panes/chart/controller/chart_connection.dart

```diff
@@ -42,10 +42,12 @@ class ChartVmConnection extends DisposableController
 
   void maybeInit() async {
     if (initialized) return;
-    initialized = true;
 
+    // We do this check for cases of accidental disconnect, in order to avoid
+    // failure for initialization of `isDeviceAndroid` next line.
```

Comment:
we don't need to worry about this. `maybeInit` is called synchronously, and we only call it after we have checked that we in fact are connected.

Reply:
Ok, added comment and assert.